app/services/validateCredentials.js

```diff
@@ -29,10 +32,11 @@ module.exports.signIn = async body => {
   const newSession = await models.session.create({ userId: user.id }).catch(() => {
     throw errors.databaseError('Could not create a new session successfully');
   });
-
-  const tokenArray = { token: newSession.id, email: user.email };
+  const current = new Date();
+  const expireTime = current.setSeconds(current.getSeconds() + parseInt(TOKEN_EXPIRATION));
+  const tokenArray = { token: newSession.id, email: user.email, expireTime };
```

Comment:
Just noticed this, but the token attribute inside the object is a bit confusing. Why not session_id or something similar?.